tests/notifications_test.py

```diff
@@ -65,7 +65,7 @@ def test_deletion_email_plaintext(snapshot, comment, snapshot_name):
 def test_invitation_email_plaintext(snapshot):
     text = render_template(
         'invitation_email.txt',
-        creator='A random cat',
+        creator='a random cat',
```

Comment:
Since names usually start with a capital letter I'd pick some name where this can be kept without looking wrong. "Catty McCatface"? :D

Reply:
Katy Purry it is

Reply:
That name is purrfect. 😹